src/docs/workflow.mdx

```diff
@@ -17,13 +17,15 @@ git pull origin HEAD --rebase
 git push origin HEAD -f
 ```
 
-**If you only need to validate CI**: add `Requires <PR link>` to the body of
-your pull request.  The `getsentry/action-get-dependent-pr` github action will
+**If you only need to validate CI**: When you create a PR add `requires <PR link>`
+to the body of it. The `getsentry/action-get-dependent-pr` github action will
 pick up this value and update the sentry revision during CI.  This is meant as
 a temporary measure to quickly validate a change -- one should commit backward
 and forward compatible changes in `sentry` since commits cannot be merged in
 lockstep.
 
+NOTE: This won't work if you edit the body of the pull request after opening it.
```

Comment:
maybe "if you realize this after opening it, edit the PR body and then push a new commit"